### PR TITLE
Improve Docs

### PR DESCRIPTION
**Pull request by frankreno**
###### Description
Improve non-helm install docs with more detailed steps and move filter/trim/relabel/custom metrics docs to standalone documentation.

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/SideBySidePrometheus.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ prometheus-operator:
       targetPort: 9200
   ...
 ```
+
+__NOTE__ To filter or add custom metrics to Prometheus, [please refer to this document](additonal_prometheus_configuration.md)
```
Comment (drduke1) [inline deploy/docs/SideBySidePrometheus.md]: "additonal" should be "additional"

File: deploy/docs/additonal_prometheus_configuration.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+Note, you need to ensure the `release` label matches the `release` label on your Prometheus pod.
+
+Detailed instructions on service monitors can be found via [Prometheus-Operator](https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md#related-resources) website.
+Once you have created this yaml file, go ahead and run `kubectl create -f name_of_yaml.yaml -n sumologic`. This will create the service monitor in the sumologic namespace.
+
+### Create a new HTTP source in Sumo Logic.
+
+To avoid [blacklisting](https://help.sumologic.com/Metrics/Understand_and_Manage_Metric_Volume/Blacklisted_Metrics_Sources) metrics should be distributed across multiple HTTP sources. You can [follow these steps](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source) to create a new HTTP source for your custom metrics. Make note of the URL as you will need it in the next step.
+
+### Update the metrics.conf FluentD Configuration
```
Comment (drduke1) [inline deploy/docs/additonal_prometheus_configuration.md]: Should "FluentD" be "Fluentd"?

File: deploy/docs/SideBySidePrometheus.md (hunk 1 above)
Comment (frankreno) [reply deploy/docs/SideBySidePrometheus.md]: Good catch, fixed

File: deploy/docs/additonal_prometheus_configuration.md (hunk 2 above)
Comment (frankreno) [reply deploy/docs/additonal_prometheus_configuration.md]: Fixed for consistency.

**Pull request closed without merging**